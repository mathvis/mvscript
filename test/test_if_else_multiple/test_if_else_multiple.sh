#!/bin/bash
helix test_if_else_multiple.mvs test_if_else_multiple.out --vsplit
