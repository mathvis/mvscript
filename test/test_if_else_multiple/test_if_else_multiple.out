Decl (Variable (VarIdentifier "a") (Just int) (Just (Type (Int 0))))
Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (Type (Int 0)))) (Block If [Decl (Assignment (AddAssign (VarIdentifier "a") (Type (Int 1))))]) (Just (ElseBlock (Block Else [Decl (Assignment (SubAssign (VarIdentifier "a") (Type (Int 1)))),Decl (IfBlock (Operation (GreaterThan (VarIdentifier "a") (Type (Int 0)))) (Block If [Decl (Assignment (AddAssign (VarIdentifier "a") (Type (Int 1))))]) (Just (ElseBlock (Block Else [Decl (Assignment (SubAssign (VarIdentifier "a") (Type (Int 1))))]))))]))))
